### PR TITLE
Finish create course feature

### PR DESCRIPTION
**Pull request by trinhducduy**
Finish create course feature

### CODE REVIEW

**Comment by luongvietdung** [comment]
ok

**Comment by trungnn142** [comment]
ok

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class CoursesController < ApplicationController
+  before_action :set_course, only: [:show, :edit, :update, :destroy]
```
Comment (dieunb) [inline app/controllers/courses_controller.rb]: don't use `set` and `get` you can use `find` or `load`

File: config/routes.rb [hunk 2]
```diff
@@ -4,4 +4,8 @@
   get "login", to: "sessions#new"
   post "login", to: "sessions#create"
   delete "logout", to: "sessions#destroy"
+
+  namespace :supervisor do
+    resources :courses
```
Comment (dieunb) [inline config/routes.rb]: supervisor root page

File: app/models/course.rb [hunk 3]
```diff
@@ -7,4 +7,11 @@ class Course < ActiveRecord::Base
 
   validates :name, presence: true
   validates :description, presence: true, length: {minimum: 100}
+  validate :start_date_cannot_be_greater_than_end_date
+
+  def start_date_cannot_be_greater_than_end_date
+    if start_date > end_date
+      errors.add(:start_date, "can't be greater than end date")
```
Comment (dieunb) [inline app/models/course.rb]: i18n

**Pull request merged**